configs/11.0/packages/openssl/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Open Secure Sockets Layer"
 ATSRC_PACKAGE_VER="1.1.0g"
-ATSRC_PACKAGE_REV=1bfb7860cb81
+ATSRC_PACKAGE_REV=7ee2a4306991
```

Comment:
OpenSSL version changed during this update (from 1.1.0g to 1.1.0h).  The relevant commit ID on OpenSSL is d4e4bd2a8163f355fa8a3884077eaec7adc75ff7.